docusaurus/docs/learn/core-concepts/pki.md

```diff
@@ -1,89 +1,56 @@
 ---
 id: pki
-title: Public Key Infrastructure (PKI)
+title: Public Key Infrastructure
 sidebar_label: PKI
 ---
 
 import PkiTroubleshootingMd from '../../guides/05-troubleshooting/pki-troubleshooting.md'
 
-All Ziti Networks leverage [Public Key Infrastructure (PKI)](https://en.wikipedia.org/wiki/Public_key_infrastructure) to
-provide secure network connections.  This page is not intended to be a comprehensive guide. What it is, is a set of
-rules that must be followed to properly configure a Ziti Network. If there are issues when connecting any portion
-of a Ziti Network to another - this page should serve as a starting point of understanding.
+[Public Key Infrastructure](https://en.wikipedia.org/wiki/Public_key_infrastructure) (PKI) is used to establish trust between all Ziti components. The Ziti controller enrolls identities with a PKI that it manages, and externally-managed PKIs may be verified with the Ziti controller and used for enrollment. Enrollment is best suited for long-lived client certificate fingerprint authentication.
 
-:::note
-This article is about managing your own PKI. There's a guide for using public CA certificates.
-:::
+Alternatively to enrollment, external PKIs may be trusted to sign authentication instruments that are used to [claim an existing Ziti identity](./security/authentication/50-external-id-claims.md).

```

Comment:
What is an "authentication instrument"?

Reply:
https://github.com/openziti/ziti-doc/pull/592/commits/7a604fb8b5d2a44a30c52b568d3480e9f0c99d73